src/Datefield.svelte

```diff
@@ -126,12 +126,14 @@
 		text = tostring(detail, format);
 		value = typeof value === 'string' ? text : clone(detail);
 		visible = false;
-		readonly ? setval(text) : focusInputElm();
+		setval(text);
+		focusInputElm();
 	}
 
 	function onfocus() {
 		inputActive = true;
 		readonly && open();
+		dispatch('focus', e);
```

Comment:
seems like the `e` parameter is missing from the function signature or am I missing something?

Reply:
@sourcegr you're right